src/app/demo/templates/templates.component.html

```diff
@@ -114,6 +114,9 @@ <h2 ngNonBindable>Event binding: (event)</h2>
       [readonly]="true"
       type="html">
     </slides-code-editor>
+    <div class="info">
+      Parentheses <b>(click)</b> are used for event binding, but "on-" can also be used <b> on-click</b>
```

Comment:
On the other hand that seems like an additional information, I would keep in in the info box, and not suggest as an alternative in the main text so that we not overflow people.